test/unit/core/simd/view_to_simd_test.cpp

```diff
@@ -89,22 +89,36 @@ class view_to_simd_test : public ::testing::Test
                                                            simd_t>;
 };
 
-using test_types = ::testing::Types<std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<int8_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<int16_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<int32_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<int64_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<uint8_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<uint16_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<uint32_t>>,
-                                    std::tuple<std::vector<seqan3::dna4>, seqan3::simd::simd_type_t<uint64_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<int8_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<int16_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<int32_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<int64_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<uint8_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<uint16_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<uint32_t>>,
-                                    std::tuple<std::deque<seqan3::dna4>, seqan3::simd::simd_type_t<uint64_t>>
```

Comment:
Can we keep this what you removed?

I would prefer if we keep `seqan3::simd::simd_type_t<>` with it's auto max-length.

Or is it really necessary to assume at least 2 elements?

Reply:
Well, if it is not 2 then the one for loop in the iterator constructor is never executed and the coverage test cries.

Reply:
I think we should fix those issues with a infrastructure change that builds for SSE4, AVX2 and so on.

I expect it right now that coverage tests and other thing related to simd are not reliable.

Reply:
as you wish